docs/snippets.rst

```diff
@@ -233,6 +233,29 @@ with a dict:
     glom('1', ({1: float, 2: int}, T.values()))
 
 
+Clamp Values
+------------
+
+A common numerical operation is to clamp values -- if they
+are above or below a certain value, assign them to that value.
+
+Using a pattern-matching glom idiom, this can be implemented
+simply:
+
+.. code-block:: python
+
+    glom(range(10), [(M < 7) | Literal(7)])
+    # [0, 1, 2, 3, 4, 5, 6, 7, 7, 7]
+
+
+What if you want to drop rather than clamp out-of-range values?
+
+.. code-block:: python
+
+    glom(range(10), [(M < 7) | Literal(SKIP)])

```

Comment:
we should make it such that it's not necessary to add Literal() around SKIP. Either by making it an instance of a type with a glomit or having M evaluation build special handling for it.

Reply:
As in, the end-user of glom should never have SKIP or STOP in their datastructures, and they should never end up with it being in the result.

Reply:
sounds good to me -- to be precise the problem here isn't SKIP being emitted in the output, it is `SKIP` not being valid as a glom

we could fix this by having

```python
def glomit(self, target, scope):
   return self
```

attached to `SKIP` somehow